flow/scripts/variables.yaml

```diff
@@ -659,10 +685,136 @@ GLOBAL_ROUTE_ARGS:
   default: -congestion_iterations 30 -congestion_report_iter_step 5 -verbose
 MATCH_CELL_FOOTPRINT:
   description: >
-    Enforce sizing operations to only swap cells that have the same layout boundary.
+    Enforce sizing operations to only swap cells that have the same layout
+    boundary.
   stages:
     - floorplan
     - place
     - cts
     - route
   default: 0
+RTLMP_MAX_LEVEL:
+  description: >
+    Maximum depth of the physical hierarchy tree.
+  default: 2
+  stages:
+    - floorplan
+RTLMP_MAX_INST:
+  description: >
+    Maximum number of standard cells in a cluster. If unset, rtl_macro_placer
+    will calculate a value based on the design attributes.
+  default: 0
```

Comment:
Do we need to give a default since unset is handled by OR and 0 isn't informative?  Likewise min/max inst/macro.

Reply:
done!